sdk/core/azure-core-test/pom.xml

```diff
@@ -95,34 +98,37 @@
       <version>1.0.0-beta.21</version> <!-- {x-version-update;com.azure:azure-core-http-vertx;current} -->
     </dependency>
 
+    <!-- Common external testing dependencies -->
+    <!-- Downstream libraries no longer need to add test dependencies on these libraries. -->
     <dependency>
       <groupId>io.projectreactor</groupId>
       <artifactId>reactor-test</artifactId>
       <version>3.4.38</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
     </dependency>
 
-    <dependency>
-      <groupId>org.apache.commons</groupId>
-      <artifactId>commons-compress</artifactId>
-      <version>1.26.0</version>  <!-- {x-version-update;org.apache.commons:commons-compress;external_dependency} -->
-    </dependency>
-
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-api</artifactId>
       <version>5.9.3</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-api;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.9.3</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-engine;external_dependency} -->
+    </dependency>
     <dependency>
       <groupId>org.junit.jupiter</groupId>
       <artifactId>junit-jupiter-params</artifactId>
       <version>5.9.3</version> <!-- {x-version-update;org.junit.jupiter:junit-jupiter-params;external_dependency} -->
     </dependency>
 
+    <!-- Used to unpack the tar.gz files of Test Proxy server -->
+    <!-- This used to be org.apache.commons:commons-compress but it was replaced with org.apache.ant:ant -->
+    <!-- This was done as ant has fewer transitive dependencies, meaning there is less chance of conflicts with other libraries. -->
```

Comment:
please add an entry to https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-test/CHANGELOG.md

Reply:
Will do in a follow-up PR alongside the azure-json update